apps/storybook/src/Breadcrumbs.test.ts

```diff
@@ -10,6 +10,7 @@ describe('Breadcrumbs', () => {
     'Folder Navigation',
     'Links',
     'Overflow',
+    'Custom Overflow',
```

Comment:
I would say to open dropdown for it to be seen on screenshot.

Reply:
Good idea, done!